GetARock/GetARock/Screens/BandDetail/BandDetailViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  BandDetailViewController.swift
+//  GetARock
+//
+//  Created by Yu ahyeon on 2023/02/13.
+//
+
+import UIKit
+
+final class BandDetailViewController: UIViewController {
+    
+    // MARK: - View
+    
+    lazy var bandTopInfoView = BandTopInfoView()
+    let bandDetailContentView = DetailContentView(type: .band)
+    
+    // MARK: - LifeCycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.addSubview(bandTopInfoView)
+        bandTopInfoView.constraint(top: self.view.topAnchor,
```

Comment:
p2;

topAnchor가 safeArea에 지정되지않아서 너무 위쪽에 딱 붙어있어요. (혹시 의도한것은 아니죠..?) safeAreaLayoutGuide 사용을 추천합니다! 

```suggestion
        bandTopInfoView.constraint(top: self.view.safeAreaLayoutGuide.topAnchor,
```

|기존|수정|
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/103009135/218933107-14431568-79f4-454e-b2ac-8656e33f9faa.png)|![image](https://user-images.githubusercontent.com/103009135/218933788-d3cf2ed2-d761-4c94-a53f-bb1e669c7651.png)|

Reply:
`BandDetailViewController`는 추후에 지도에서 밴드를 선택하면 모달로 나오게 될 친구라서
일부로 AutoLayout의 TOP의 값을 0으로 준 게 맞습니다!

<img width="250" alt="image" src="https://user-images.githubusercontent.com/103012104/218987341-17330e64-90b9-457f-9007-fdd828729d3e.png">

Reply:
설마가 맞앗군요 ㅋㅋㅋㅋㅋㅋㅋ